CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Error handling for when API response cannot be parsed as JSON ([#174s](https://github.com/forcedotcom/sf-fx-runtime-nodejs/pull/174))
```

Comment:
```suggestion
- Error handling for when API response cannot be parsed as JSON ([#174](https://github.com/forcedotcom/sf-fx-runtime-nodejs/pull/174))
```

Reply:
This was marked as resolved but the typo is still there. Maybe it was over-written when force pushed?

You can prevent that with:

```
git push --force-with-lease
```
I have an alias for that:

```
git config --global alias.pushf "push --force-with-lease"
```

So i can `git pushf` and it does the same thing by default.